scripts/lang/ca.py

```diff
@@ -85,15 +87,53 @@
     "marca-nocat": "term(lookup_italic(concat(parts, sep=', ', indexes=[2, 3, 4, 5]), 'ca'))",
     # {{q|tenir bona planta}}
     "q": "term(parts[-1])",
-    # {{terme|it|come}}
-    "terme": "parts[-1]",
 }
 
 # Templates that will be completed/replaced using custom style.
 templates_other = {
     "m": "m.",
 }
 
+
+def last_template_handler(template: Tuple[str, ...], locale: str) -> str:
+    """
+    Will be called in utils.py::transform() when all template handlers were not used.
+        >>> last_template_handler(["terme", "it", "come"], "ca")
```

Comment:
Add a blank line to separate the description from doctests.

Reply:
OK. Done.